homeassistant/components/nanoleaf/manifest.json

```diff
@@ -7,7 +7,7 @@
   "zeroconf": ["_nanoleafms._tcp.local.", "_nanoleafapi._tcp.local."],
   "homekit" : {
     "models": [
-      "NL*"
+      "NL29", "NL42", "NL52"
```

Comment:
The Nanoleaf Lines are missing here. I don't know the model number, and it isn't in the API documentation. I'll try to find.

Reply:
Looking at https://nanoleaf.me/en-GB/products/, Elements are NL52, NL42 are Hexagon, and NL29 are Canvas. So Line must be NL59?

Reply:
The product model does not always match the HomeKit model. I don't have a better idea, so we have to assume that the model matches the HomeKit model in this case.

Reply:
I've got a few contacts in Nanoleaf, I'll ask politely & see if they'll give me a mapping. For now I'll match the SKU.

Reply:
Added Lines, Triangles, Mini Triangles. Not sure on Light Panels, there are a lot of different NL## SKUs.

Reply:
Triangles and Mini triangles use NL42 (Hexagons) for SSDP, I don't know if this is also the case for HomeKit but adding a few models too much is not a problem, so for me this is okay. :+1: 
Thanks! :smile: